.github/scripts/generate_binary_build_matrix.py

```diff
@@ -178,11 +178,7 @@ def generate_wheels_matrix(os: str,
     if python_versions is None:
         # Define default python version
         python_versions = list(FULL_PYTHON_VERSIONS)
-
-        if os == "linux":
-            # NOTE: We only build 3.11 wheel on linux as 3.11 is not
-            # available on conda right now
-            python_versions.append("3.11")
+        python_versions.append("3.11")
```

Comment:
Does not this mean we can change FULL_PYTHON_VERSIONS to include 3.11?

Reply:
No, because we still have to dance this dance for conda, where it is available only for Linux at the moment. When I'm ready to enable those for Windows, I'll post update the full version.